src/pudl/transform/eia860.py

```diff
@@ -579,6 +579,7 @@ def _make_phone_number(col1, col2, col3):
             entity_type=lambda x: x.entity_type.map(ENTITY_TYPES)
         )
         .pipe(pudl.helpers.convert_to_date)
+        .fillna({'entity_type': pd.NA})
```

Comment:
This changes np.nans to pd.NAs. There are no "unknown" in the EIA 860 entity types for all years.

Reply:
I know this field comes up a lot in the EIA-861, so I wonder if this is where the `unknown` and `other` values are showing up? I'm working on compiling that metadata right now, and for some reason 100% of the `entity_type` column in the `sales_eia861` table is currently showing up as null.

But if this all works for the existing EIA related DB tables then that's fine.

Reply:
I'm going to have to do some messing around with `entity_type` in the EIA-861 (which is where it mostly shows up) and might end up cleaning up the categories. It looks like they use two different sets of codes in there. The short codes and full names with capitalization, which is why I'm getting fully null columns.